internal/codespaces/api/api.go

```diff
@@ -168,17 +168,19 @@ func (a *API) GetRepository(ctx context.Context, nwo string) (*Repository, error
 
 // Codespace represents a codespace.
```

Comment:
Can you add a link to the place where the meaning of all of these fields is specified?

Reply:
You mean add a link to the API docs? The fields I added in this PR aren't publicly documented yet since they are still behind a feature flag. I can add a link to the public API docs, but it won't have the two new fields in it.